Dockerfile.mysql

```diff
@@ -2,10 +2,10 @@
 FROM mysql:latest
 
 # Environment variables for MySQL configuration
-ENV MYSQL_ROOT_PASSWORD=1234
-ENV MYSQL_DATABASE=app_db
-ENV MYSQL_USER=app_user
-ENV MYSQL_PASSWORD=1234
+ENV MYSQL_ROOT_PASSWORD="default"
+ENV MYSQL_DATABASE="default"
+ENV MYSQL_USER="default"
```

Comment:
The environment variable `MYSQL_USER` is set to 'default'. Ensure this is the intended value or replace it with the actual username required for your application.